### PR TITLE
LPS-121921 Migrate v2 usages of Management Toolbar in user-associated-data-web

### PR DESCRIPTION
**Pull request by kresimir-coko**
First time migrator here 👋 
I tested everything locally, made minimal changes and everything seem to work just as it did before, with an addition of better selected item information in the `add_uad_export_processes.jsp` page - showing the number of selected and total number as well as the `Clear` and `Select All` buttons which weren't present in the `v2` variant.

## Testing

1. Add a new user with the `Administrator` role
Note: You need to add `company.security.strangers.verify=false` to your `portal-ext.properties` if you're asked for a validation token
2. Log into that new user and create some content (blog, bookmark, etc.)
3. Log back into your Test user
4. Go back to Users and Administration, click on the actions menu for your new user and click `Export Personal Data`
5. Add content types that you created as the temporary user
Note: When choosing what content to export you can find the `add_uad_export_processes.jsp` file to test.
6. When you add the content types you'll be taken to the page with a list of exported processes
Note: Here you can test the `view_uad_export_processes.jsp` file.
7. Go one step back, to the view of the Users and Organisations page, click on the actions menu and `Delete Personal Data`.
Note: This leads you to a page where you can test the `view_uad_entities.jsp`.

### CODE REVIEW

File: modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/add_uad_export_processes.jsp [hunk 1]
```diff
@@ -60,7 +60,7 @@ renderResponse.setTitle(StringBundler.concat(selectedUser.getFullName(), " - ",
 				}
 				%>
 
-				<clay:management-toolbar-v2
+				<clay:management-toolbar
 					disabled="<%= disableManagementBar %>"
 					itemsTotal="<%= uadApplicationExportDisplayList.size() %>"
 					namespace="<%= liferayPortletResponse.getNamespace() %>"
```
Comment (ambrinchaudhary) [inline modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/add_uad_export_processes.jsp:66]: We can simply remove the `namespace` attribute here, is is set directly in the Tag.

**Review by ambrinchaudhary — CHANGES_REQUESTED** [review]
Just a minor change requested, otherwise LGTM!

File: modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/add_uad_export_processes.jsp (hunk 1 above)
Comment (kresimir-coko) [reply modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/add_uad_export_processes.jsp:66]: 👍

**Review by julien — APPROVED** [review]

**Comment by julien** [comment]
@kresimir-coko LGTM, was interesting testing this.

**Comment by julien** [comment]
@kresimir-coko do you know who we need to forward this to?
(I'd be happy if we could just forward it ourselves seeing it's a pretty simple change, but I thought I'd ask)

**Comment by julien** [comment]
Replying to myself: seems it's going to be for User and System management.

**Review by markocikos — COMMENTED** [review]
@kresimir-coko  One minor change to do, looks good otherwise!

File: modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/view_uad_entities.jsp [hunk 2]
```diff
@@ -36,7 +36,7 @@ if (parentContainerId > 0) {
 long[] groupIds = viewUADEntitiesDisplay.getGroupIds();
 %>
 
-<clay:management-toolbar-v2
+<clay:management-toolbar
 	displayContext="<%= new ViewUADEntitiesManagementToolbarDisplayContext(request, liferayPortletRequest, liferayPortletResponse, viewUADEntitiesDisplay) %>"
```
[the remark this answers is not in the document]
Comment (kresimir-coko) [reply modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/view_uad_entities.jsp]: Fixed in https://github.com/liferay-frontend/liferay-portal/pull/877/commits/37b4e0da059ab5e54020b21a7281b779d227baa2

**Comment by kresimir-coko** [comment]
@julien 
> Replying to myself: seems it's going to be for User and System management.

Do we know who to send the PR in that team? I don't think I've ever worked on anything belonging to that team before.

**Comment by julien** [comment]
@kresimir-coko yes we do. In this case, it should be forwarded to @drewbrokke

**Comment by kresimir-coko** [comment]
@julien is there a way to `ci:forward` to him, or the old fashioned way of directly sending him a PR?

**Comment by julien** [comment]
@kresimir-coko I usually just send the PR manually, opening this kind of URL in my browser

```
https://github.com/THEIR_USER/liferay-portal/compare/master...YOUR_USER:BRANCH
```

**Comment by wincent** [comment]
> @julien is there a way to `ci:forward` to him, or the old fashioned way of directly sending him a PR?

No @kresimir-coko, [you cannot have nice things](https://liferay.slack.com/archives/C19PWQ74J/p1615227886065600).

**Comment by jbalsas** [comment]
I see @julien's approval here and @markocikos concerns addressed, so going to forward this to @drewbrokke

**Comment by jbalsas** [comment]
@kresimir-coko actually already forwarded this to @drewbrokke at https://github.com/drewbrokke/liferay-portal/pull/1327 although it's missing the lovely description it has in here... @kresimir-coko, when you have the chance, please be so kind as to add it in there as well, surely Drew will appreciate more details without the need to jump through links 😉

**Pull request closed without merging**